packages/market-matcher/src/MatchableAgreement.ts

```diff
@@ -25,19 +23,9 @@ export class MatchableAgreement {
             .result();
     }
 
-    public async missingEnergyForDemand(demand: Demand.IDemand, config: Configuration.Entity) {
-        const { start, timeframe } = this.agreement.offChainProperties;
-        const currentPeriod = await Utils.getCurrentPeriod(start, timeframe, config);
-
+    public async missingEnergyForDemand(demand: Demand.IDemand) {
         const { targetWhPerPeriod } = demand.offChainProperties;
```

Comment:
Are we tracking the missing energy using `targetWhPerPeriod` ?

Reply:
Tbh I just removed this because I was removing separate references to matchers for every agreement - thought we will be refactoring later in the sprint.

Reply:
And I guess this build passed because there is no test written that checks if the demand size decreases. This should be fixed.